common-files/constants/constants.json

```diff
@@ -0,0 +1,28 @@
+{
+  "CONTRACT_ARTIFACTS": "/app/build/contracts",
+  "PROPOSERS_CONTRACT_NAME": "Proposers",
+  "SHIELD_CONTRACT_NAME": "Shield",
+  "CHALLENGES_CONTRACT_NAME": "Challenges",
+  "STATE_CONTRACT_NAME": "State",
+  "BLOCK_PROPOSED_EVENT_NAME": "BlockProposed",
+  "EXCLUDE_DIRS": "common",
+  "PROOF_QUEUE": "generate-proof",
+  "MAX_QUEUE": 5,
+  "ZKP_KEY_LENGTH": 32,
+  "CONFIRMATION_POLL_TIME": 1000,
+  "CONFIRMATIONS": 12,
+  "NODE_HASHLENGTH": 32,
+  "DEFAULT_ACCOUNT_NUM": 10,
+  "ZERO": "0x0000000000000000000000000000000000000000000000000000000000000000",
+  "HASH_TYPE": "poseidon",
```

Comment:
Note a constant - we change the hash type from time to time

Reply:
moved to config